test/functional/editor/macro_spec.lua

```diff
@@ -11,9 +11,11 @@ local meths = helpers.meths
 local insert = helpers.insert
 local curbufmeths = helpers.curbufmeths
 
-before_each(clear)
```

Comment:
The `reg_recorded()` describe block at the bottom of this file now needs a `before_each(clear)`, otherwise it will fail when run alone.

Reply:
Yes, added. Thanks for the catch :)